pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java

```diff
@@ -166,6 +166,12 @@ private static boolean argsContains(String[] args, String arg) {
                     "c-" + brokerConfig.getClusterName()
                         + "-fw-" + hostname
                         + "-" + workerConfig.getWorkerPort());
+                // inherit broker authorization setting
+                workerConfig.setAuthorizationEnabled(brokerConfig.isAuthorizationEnabled());
+                workerConfig.setAuthorizationProvider(brokerConfig.getAuthorizationProvider());
+
+                workerConfig.setZooKeeperSessionTimeoutMillis(brokerConfig.getZooKeeperSessionTimeoutMillis());
```

Comment:
Doesn't This violate some abstraction layers here?
The zkoptimeout is only relevant for default pulsar provider right? In reality this should be basically a call to inherit all the config relevant to that particular auth provider?

Reply:
There is no other way to specify these configs at this time. Currently, there isn't any separation between Pulsar configs and auth configs. Perhaps, we should have configs on a per Authorization/Authentication provider impl basis, but currently there is no such thing.